##################################################################################
$Id$
##################################################################################

			     SDL PATCH REPORT
			     ================

SDL-Release: 1.2.5
Patch-ID: SDL-giffordj-001

Patch by       :    giffordj@linkline.com

Problem-Description: Fixes for GCC 3.3 Errors

This patch is a based of suggested patches posted found on the web. This
is not an offical patch for SDL. But it works for me.

Patch:

diff -uNr SDL-1.2.5.orig/src/video/SDL_stretch.c SDL-1.2.5/src/video/SDL_stretch.c
--- SDL-1.2.5.orig/src/video/SDL_stretch.c	2002-03-06 03:23:03.000000000 -0800
+++ SDL-1.2.5/src/video/SDL_stretch.c	2003-05-16 17:15:17.000000000 -0700
@@ -261,9 +261,8 @@
 			break;
 		    default:
 #ifdef __GNUC__
-			__asm__ __volatile__ ("
-				call _copy_row
-			"
+			__asm__ __volatile__ (
+			"call _copy_row"
 			: "=&D" (u1), "=&S" (u2)
 			: "0" (dstp), "1" (srcp)
 			: "memory" );
